CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.13.4
+
+## Enhancements
+
+- Added support for `externalDocs`.
+- Parser will not give warning about unsupported collection formats in parameters.
```

Comment:
Why parser won't warn about them if they're unsupported?

Reply:
Oh I think this is a mistake in changelog the following warning is emitted:

> Parameters of collection format '${format}' are not supported

Reply:
Yup, it was mistype of `now` to `not`.